ui/main/src/app/modules/businessconfigparty/iframedisplay.component.ts

```diff
@@ -68,6 +68,10 @@ export class IframeDisplayComponent implements OnInit, OnDestroy {
         });
     }
 
+    private removeEncodedParams(url: string): string {
+        return decodeURIComponent(url).split('?')[0];
+    }
```

Comment:
This method is used only to remove encoded characters from the menu_entry_id , while do we need to split it ? 

I suggest to remove this method and add directly before : 
```
 .queryMenuEntryURL(this.menu_id, decodeURIComponent(this.menu_entry_id))
```